src/io/pti.jl

```diff
@@ -943,4 +943,744 @@ function _populate_defaults!(data::Dict)
+    # Try to get the value from the PM component (is in lower case), else use the default
+    for (key, default) in pti_default
+        if ! haskey(sub_data, key)
+            sub_data[key] = get(pm_comp, lowercase(key), default)
+        end
+    end
+end
+
+
+"""
+Create a header for the case
+"""
+function _pm2psse_header(pm::Dict{String, Any})
+    sub_data = Dict{String, Any}()
+
+    for (dtype, _) in _transaction_dtypes
+        sub_data[dtype] = get(pm, dtype, _default_case_identification[dtype])
+    end
+
+    return sub_data
+end
+
+
+"""
+Parses PM Bus data to PSS(R)E-style.
+"""
+function _pm2psse_bus(pm_bus::Dict{String, Any})
+    sub_data = Dict{String, Any}()         
+    sub_data["I"] = pm_bus["bus_i"]
+
+    if haskey(pm_bus, "name")
+        sub_data["NAME"] = "\'$(pm_bus["name"])\'"
+    elseif haskey(pm_bus, "string")
+        sub_data["NAME"] = "\'$(pm_bus["string"])\'"
+    else
+        sub_data["NAME"] = "\'$(_default_bus["NAME"])\'"
+    end
+
+    sub_data["BASKV"] = get(pm_bus, "base_kv", _default_bus["BASKV"])
+    sub_data["IDE"] =  get(pm_bus, "bus_type", _default_bus["IDE"])
+    sub_data["AREA"] = get(pm_bus, "area", _default_bus["AREA"])
+    sub_data["ZONE"] = get(pm_bus, "zone", _default_bus["ZONE"])
+    sub_data["OWNER"] = get(pm_bus, "owner", _default_bus["OWNER"])
+    sub_data["VM"] = get(pm_bus, "vm", _default_bus["VM"])
+    sub_data["VA"] = get(pm_bus, "va", _default_bus["VA"])
+    sub_data["NVHI"] = get(pm_bus, "vmax", _default_bus["NVHI"])
+    sub_data["NVLO"] = get(pm_bus, "vmin", _default_bus["NVLO"])
+
+    _export_remaining!(sub_data, pm_bus, _pti_defaults["BUS"])
+    
+    return sub_data
+end
+
+
+"""
+Parses PM Load data to PSS(R)E-style.
+"""
+function _pm2psse_load(pm_load::Dict{String, Any}, area::Int64, owner::Int64, zone::Int64)
+    
+    sub_data = Dict{String, Any}()
+    sub_data["I"] = pm_load["load_bus"] 
+    sub_data["ID"] = "\'$(pm_load["source_id"][end])\'" # CHECK WITH DICTS FROM MATPOWER
+    sub_data["STATUS"] = get(pm_load, "status", _default_load["STATUS"])
+    sub_data["PL"] = get(pm_load, "pd", _default_load["PL"])
+    sub_data["QL"] = get(pm_load, "qd", _default_load["QL"])
+    sub_data["AREA"] = get(pm_load, "area", area)
+    sub_data["OWNER"] = get(pm_load, "owner", owner)
+    sub_data["ZONE"] = get(pm_load, "zone", zone)
+
+    _export_remaining!(sub_data, pm_load, _pti_defaults["LOAD"])
+
+    return sub_data
+end
+
+
+"""
+Parses PM fixed shunt to PSS(R)E-style
+"""
+function _pm2psse_fixed_shunt(pm_shunt::Dict{String, Any})
+    sub_data = Dict{String, Any}()
+    sub_data["I"] = pm_shunt["shunt_bus"] # Not defaul allowed
+    sub_data["ID"] = "\'$(pm_shunt["source_id"][end])\'"
+    sub_data["STATUS"] = get(pm_shunt, "status", _default_fixed_shunt["STATUS"])
+    sub_data["GL"] = get(pm_shunt, "gs", _default_fixed_shunt["GL"])
+    sub_data["BL"] = get(pm_shunt, "bs", _default_fixed_shunt["BL"])
+    
```

Comment:
no `_export_remaining!`?

Reply:
There is no more to export. From the PSSE docs:

> Each fixed bus shunt data record has the following format:
I, ID, STATUS, GL, BL

I can put `_export_remaining!` and delete `sub_data["STATUS"]`

Reply:
no problem, this is fine as is.